src/main/java/com/ibm/devops/connect/CloudSocketComponent.java

```diff
@@ -210,7 +210,9 @@ public void handleDelivery(String consumerTag, Envelope envelope,
                             String encodedString = getEncodedString(plainCredentials);
                             String authorizationHeader = "Basic " + encodedString;
                             String rootUrl = Jenkins.getInstance().getRootUrl();
-                            HttpResponse<String> response = Unirest.get(rootUrl+"job/"+jobName.replaceAll("/", "/job/").replaceAll(" ", "%20")+"/lastBuild/consoleText")
+                            String path = "job/"+jobName.replaceAll("/", "/job/")+"/lastBuild/consoleText";
```

Comment:
I'm not seeing how any special chars in `jobName` (such as spaces) would get properly URL escaped. Also would be better to use a URL builder to construct it rather than just string concatenation?

Reply:
@aboe026 Made changes, added URL Builder for better handling of special chars.

Reply:
> I'm not seeing how any special chars in `jobName` (such as spaces) would get properly URL escaped. Also would be better to use a URL builder to construct it rather than just string concatenation?

@aboe026 Made changes, Added URL Builder for better handling of special chars